controllers/depresolver/depresolver_config.go

```diff
@@ -94,6 +94,9 @@ func (dr *DependencyResolver) validateConfig(config *Config) (err error) {
 	if config.Log.Format == NoFormat {
 		return fmt.Errorf("invalid '%s', allowed values ['','%s','%s']", LogFormatKey, JSONFormat, SimpleFormat)
 	}
+	if config.EdgeDNSType == DNSTypeError {
+		return fmt.Errorf("more than one EdgeDNS types recognised")
```

Comment:
Spelling fiix
```suggestion
		return fmt.Errorf("More than one EdgeDNS provider recognized")
```

Reply:
Error string should not be capitalized, otherwise IDE prints warning, see: https://github.com/golang/go/wiki/CodeReviewComments#error-strings. 

Otherwise, fixing `provider recognized `

Reply:
Ah, right, this is error message, resolved.